### PR TITLE
Create MergeTools.py

### PR DESCRIPTION
**Pull request by HighnessAtharva**
Solution to Merge the tools exercise was missing. Added the same.

### CODE REVIEW

File: Strings/MergeTools [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Subdomain : Strings
+Domain    : Python
+Author    : Atharva Shah
+Created   : 15 March 2021
+Problem   : https://www.hackerrank.com/challenges/merge-the-tools/
+'''
+from collections import OrderedDict
+
+def merge_the_tools(string, k):
+    for i in range(0, len(string), k):
```
Comment (arsho) [inline Strings/MergeTools:12]: Your solution passes all the cases. But I think we can change this line to: `for i in range(0, len(string)-k+1, k):`

**Review by arsho — COMMENTED** [review]
Dear Artharva,

Thank you for adding the problem. Please check the review. 
**Additional request:** 
Please add this problem name in the `HackerrankUtility/python_info.txt` file inside the `Strings` section.

File: Strings/MergeTools (hunk 1 above)
Comment (HighnessAtharva) [reply Strings/MergeTools]: Done. Yes, this is certainly better.

**Review by arsho — APPROVED** [review]
Thank you for the update

**Pull request merged**